docs/tutorials/projects/verification-system.md

```diff
@@ -0,0 +1,240 @@
+
+{{< alert title="Requirements" color="note" >}}
+Before configuring your camera, you must [create a robot](/fleet/machines/#add-a-new-robot).
+{{< /alert >}}
+
+## Configure a camera
+
+Navigate to the **Config** tab of your machine's page on the [Viam app](https://app.viam.com).
+Configure the camera you want to provide an image stream for your verification system on.
+
+We configured ours as a webcam named `"my-webcam"`.
+Follow [these instructions](/build/configure/components/camera/webcam/) to do so.
+
+## Configure an `mlmodel` person detector
+
+Create a detector to detect when people are in your verification system camera's field of vision:
+
+- [Capture images from your camera](/data/capture/#configure-data-capture-for-individual-components) with Viam's data management service, then [create a dataset](/data/dataset/#create-a-dataset-and-add-data).
+- Label your data with [bounding boxes](/data/dataset/#bounding-boxes) labeled as `Person` where a person is present to create an object detection model for people.
+- [Train a model on your dataset](/ml/train-model/).
+- [Deploy the model](/ml/deploy/) to your machine.
+
+Now that you've trained and deployed the model itself, [configure an `mlmodel` detector model](/ml/vision/detection/#configure-an-mlmodel-detector) with the name of the ML model service you deployed your model as.
+Name this detector `"people-detect"`.
+
+For example, we configured a person detector as follows:
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "type": "mlmodel",
+    "model": "tflite_cpu",
+    "attributes": {
+      "model_path": "${packages.persondetect0tflite}/persondetect0.tflite",
+      "label_path": "${packages.persondetect0tflite}/persondetectlabels.txt",
+      "num_threads": 1
+    },
+    "name": "persondetect"
+  },
+  {
+    "name": "people-detect",
+    "type": "vision",
+    "model": "mlmodel",
+    "attributes": {
+      "mlmodel_name": "persondetect"
+    }
+  }
+]
+```
+
+## Configure a facial detector
+
+To create a detector that can recognize individual faces, use Viam Lab's `facial-detector` module, available in the [registry](https://app.viam.com/module/viam-labs/facial-detector).
+This is a modular vision service that uses the DeepFace library to perform facial detections.
+
+- Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/).
+  Click on the **Services** subtab and click **Create service**.
+  Select the `vision` type, then select the `detector:facial-detector` model. Enter a name for your vision service and click **Create**.
+- Edit the attributes as applicable according to the configuration information on [GitHub](https://github.com/viam-labs/facial-detection). Label a photo of the face of each person you want your security system to recognize in `"face_labels"`.
+
+Name this detector `"face-detect"`.
+
+For example, we configured a facial detector as follows:
```

Comment:
I think you'll need to explain the attributes. I have no idea what those are for and whether they relate to the model i trained and deployed. You don't need to recreate the readme docs but a little more info on what's happening is needed

Reply:
Added more info